pkg/clustermesh/internal/remote_cluster.go

```diff
@@ -27,6 +27,11 @@ import (
 	"github.com/cilium/cilium/pkg/option"
 )
 
+var (
+	remoteConnectionControllerGroup = controller.NewGroup("clustermesh-remote-etcd")
```

Comment:
```suggestion
	remoteConnectionControllerGroup = controller.NewGroup("clustermesh-remote-cluster")
```

Nit., as it does not only handle the etcd connection.